ipamd/ipamd.go

```diff
@@ -509,7 +509,10 @@ func (c *IPAMContext) tryUnassignIPsFromAll() {
 			// Delete IPs from datastore
 			var deletedIPs []string
 			for _, toDelete := range ips {
-				err := c.dataStore.DelIPv4AddressFromStore(eniID, toDelete)
+				// Don't force the delete, since a freeable IP might have been assigned to a pod
+				// before we get around to deleting it.
+				const force = false
```

Comment:
Instead of this, a more idiomatic way of using "named params" in a function signature is to do the following:

```go
err := c.dataStore.DelIPv4AddressFromStore(eniID, toDelete, false /* force */)
```

Reply:
Sure, done.